src/protocols/secure_channel/tests/TestDefaultSessionResumptionStorage.cpp

```diff
@@ -15,15 +15,17 @@
  *    limitations under the License.
  */
 
+#include <lib/support/DefaultStorageKeyAllocator.h>
+#include <lib/support/TestPersistentStorageDelegate.h>
 #include <lib/support/UnitTestRegistration.h>
 #include <nlunit-test.h>
 
-#include <lib/support/TestPersistentStorageDelegate.h>
-
 // DefaultSessionResumptionStorage is a partial implementation.
 // Use SimpleSessionResumptionStorage, which extends it, to test.
 #include <protocols/secure_channel/SimpleSessionResumptionStorage.h>
 
+#define ARRAY_SIZE(_array) ((sizeof(_array) / sizeof(_array[0])))
```

Comment:
We already have an ArraySize macro for this, fwiw, in `src/lib/support/CodeUtils.`

Reply:
Was not aware.  Can do a fast-follow cleanup PR.

Thank you.

Reply:
PR here: https://github.com/project-chip/connectedhomeip/pull/23110